packages/contracts/deploy/new/00_managing-dao/00_managing-dao.ts

```diff
@@ -1,34 +1,39 @@
 import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
Changes look good, but does the deployment script need to be changed on this PR?